src/main/java/Todo.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Represents a todo task in the chatbot application.
+ * A todo task is a basic task type without any date or time constraints.
+ */
+public class Todo extends Task{
+    /**
+     * Constructs a new Todo task with a specified name and task number.
+     *
+     * @param name The name or description of the todo task.
+     * @param taskNo The unique number assigned to the task.
+     */
+    public Todo(String name, int taskNo) {
+        super(name, taskNo);
+    }
+
+    /**
+     * Marks this todo task as completed.
+     * Overrides the markedTask method in the Task class to include todo-specific details in the confirmation message.
+     */
+    @Override
+    public void markedTask() {
+        this.hasDone=true;
+        System.out.println("    " + "--------------");
+        System.out.println("    " + "Nice! I've marked this task as done:");
+        System.out.println("      [T]" + "[X] "+this.name);
+        System.out.println("    " + "--------------");
```

Comment:
Do you think you could somehow refactor this as well to utilize the Task class's markedTask()? Same for unmarkedTask and the Event and Deadline classes